.github/workflows/ci.yml

```diff
@@ -111,9 +111,17 @@ jobs:
       - name: Check library missing peers
         run: npx skyux-dev check-lib-missing-peers
       - name: Check library resources
-        run: npx skyux-dev check-lib-resources
+        run: |
+          # Get the affected projects.
+          AFFECTED_PROJECTS=$(npx nx print-affected -t build --exclude=storybook,e2e-schematics,prettier-schematics --select=tasks.target.project --type=lib)
```

Comment:
Getting the affected projects is already a feature of `dev-infra`. Perhaps we should modify the `check-lib-resources` command to default to affected projects, instead of passing them into the `--projects` argument?